.github/workflows/black_python_formatter.yml

```diff
@@ -0,0 +1,42 @@
+name: Black Python Formatter
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    # Name the Job
+    name: Formatter
```

Comment:
May want to be more specific, as we'll no doubt have linters for other languages (TS, even HCL)